docs/source/error_code_list2.rst

```diff
@@ -236,13 +236,12 @@ incorrect control flow or conditional checks that are accidentally always true o
 Check that imported or used feature is deprecated [deprecated]
 --------------------------------------------------------------
 
-By default, mypy generates a note if your code imports a deprecated feature explicitly with a
+If you use :option:`--enable-error-code deprecated <mypy --enable-error-code>`,
+mypy generates an error if your code imports a deprecated feature explicitly with a
 ``from mod import depr`` statement or uses a deprecated feature imported otherwise or defined
 locally.  Features are considered deprecated when decorated with ``warnings.deprecated``, as
-specified in `PEP 702 <https://peps.python.org/pep-0702>`_.  You can silence single notes via
+specified in `PEP 702 <https://peps.python.org/pep-0702>`_.  You can silence single errors via
 ``# type: ignore[deprecated]`` or turn off this check completely via ``--disable-error-code=deprecated``.
```

Comment:
The "or turn off this check..." part of the sentence could also be removed.

Reply:
Thanks! If we decide to go through with this change, I'll take another look at this.